atlp-rwanda/rca-phantom-team1-bn
# Feat/buses crud

## What does this PR do
- It implements the bus feature
- It adds the feature tests

## Description
- Operator is able to perform the CRUD operations on buses
- Buses can be seeded at once for development purposes by running `npm run db:seeds`

## How to manually test this 
- Check in the branch `feat/buses-crud`
- Run the server
- Open the Swagger API documentation to test the buses

## Relevant Trello story 
[11-crud-operations-for-buses][(https://trello.com/c/a6wcZ3PE/9-create-roles-and-set-permissions)](https://trello.com/c/maUQgDhP/11-crud-operations-for-buses)

## Screenshots
![image](https://user-images.githubusercontent.com/53254885/227944406-db62114f-9ac7-4c8e-bad3-2dc9f7ff8317.png)

![image](https://user-images.githubusercontent.com/53254885/227944593-bc36c327-a893-4eb3-8f93-48a0f75cd073.png)
